starter_code/src/movies.js

```diff
@@ -1,20 +1,167 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
 
+// Turn duration of the movies from hours to minutes
+function turnHoursToMinutes(movies) {
+    return movies.map(function (curMovie) {
```

Comment:
Overly complicated. I like that you took very thorough approach but it might have xyz different combinations so this is not very efficient. Take a look at the solution Nick gave to you.